mifospay/src/main/res/layout/fragment_kyc.xml

```diff
@@ -1,6 +1,181 @@
+        <android.support.v7.widget.CardView
+            android:id="@+id/cv_lvl2"
+            android:layout_width="125dp"
+            android:layout_marginRight="72dp"
+            android:layout_marginTop="10dp"
+            android:layout_height="60dp"
+            android:layout_marginBottom="@dimen/value_30dp"
+            app:cardCornerRadius="@dimen/value_10dp">
+
+            <LinearLayout
+                android:id="@+id/ll_lvl2"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@drawable/cardview_round_blue"
+                android:orientation="vertical"
+                android:padding="@dimen/value_10dp">
+
+                <TextView
+                    android:id="@+id/lvl2text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="@dimen/value_10dp"
+                    android:text="@string/level_2"
+                    android:textAlignment="center"
+                    android:textColor="@color/white"
+                    android:textSize="@dimen/value_16sp" />
+
+                <!--<TextView-->
+                <!--android:layout_width="wrap_content"-->
+                <!--android:layout_height="wrap_content"-->
+                <!--android:layout_marginBottom="@dimen/value_5dp"-->
+                <!--android:layout_marginLeft="@dimen/value_10dp"-->
+                <!--android:layout_marginRight="@dimen/value_10dp"-->
+                <!--android:layout_marginTop="@dimen/value_10dp"-->
+                <!--android:text="@string/level_2_description"-->
+                <!--android:textColor="@color/gray"/>-->
+
+            </LinearLayout>
+        </android.support.v7.widget.CardView>
+
+            <TextView
+                android:id="@+id/completed2"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/value_30dp" />
+        </LinearLayout>
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+        <android.support.v7.widget.CardView
+            android:id="@+id/cv_lvl3"
+            android:layout_width="125dp"
+            android:layout_marginRight="72dp"
+            android:layout_marginTop="10dp"
+            android:layout_height="60dp"
+            android:layout_marginBottom="@dimen/value_30dp"
+            app:cardCornerRadius="@dimen/value_10dp">
+
+            <LinearLayout
+                android:id="@+id/ll_lvl3"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@drawable/cardview_round_blue"
+                android:orientation="vertical"
+                android:padding="@dimen/value_10dp">
+
+                <TextView
```

Comment:
Did you just use CardViews instead of Buttons with 

style="@style/Widget.MaterialComponents.Button" or (...`Buton.OutlinedButton`)

I've told you about it earlier and you didn't even listen to my advices :|

Reply:
@letelete  card views were used in the master branch of the app  so I used them in the redesign as well .What exactly do you want me to use ?